app/src/test/java/io/intrepid/contest/screens/contestcreation/namecontest/NameContestPresenterTest.java

```diff
@@ -18,13 +21,14 @@
     NameContestContract.View mockView;
     @Mock
     Contest.Builder mockContestBuilder;
-    private NameContestContract.Presenter nameContestPresenter;
+    private NameContestPresenter nameContestPresenter;
 
     @Before
     public void setup(){
         MockitoAnnotations.initMocks(this);
-        nameContestPresenter = new NameContestPresenter(mockView, TestPresenterConfiguration.createTestConfiguration(),
-                                                        mockContestBuilder);
+        nameContestPresenter = spy(new NameContestPresenter(mockView,
```

Comment:
I was using spy before when trying to find a workaround to test the static methods. I'm going to revert this though.